src/ConversationList/ConversationListBox.vala

```diff
@@ -425,4 +451,62 @@ public class Mail.ConversationListBox : VirtualizingListBox {
     public void undo_expired () {
         move_handler.expire_undo ();
     }
+
+    private bool create_context_menu (Gdk.Event e, ConversationListItem row) {
+        var item = (ConversationItemModel)row.model_item;
+
+        var menu = new Gtk.Menu ();
+
+        var delete_menu_item = new Gtk.MenuItem.with_label (_("Delete"));
+        menu.add (delete_menu_item);
```

Comment:
It would be great to show the accelerator labels in the context menu as well to make keyboard shortcuts discoverable. Granite's `AccelLabel` comes in handy for this.

See the code of the HeaderBar for an example:
https://github.com/jhaygood86/mail/blob/392b9651b79ab6fd1f4bbc9e7068e88f45162763/src/HeaderBar.vala#L115-L118